### PR TITLE
Add puppet-agent plugin

### PR DESCRIPTION
**Pull request by jan-win1993**
https://github.com/opnsense/plugins/pull/2358

### CODE REVIEW

**Comment by fraenki** [comment]
@fichtner This is not in 21.1.6 yet, right?

**Comment by fichtner** [comment]
Yes, not enough time if we wanted to release 21.1.6 this week.

**Comment by fichtner** [comment]
(maybe you have some time to talk on IRC, second day back at work for me)

File: config/21.1/ports.conf [hunk 1]
```diff
@@ -221,6 +221,7 @@ sysutils/munin-node				arm,arm64
 sysutils/node_exporter				arm,arm64
 sysutils/nut					arm,arm64
 sysutils/pftop
+sysutils/puppet7
```
Comment (fichtner) [inline config/21.1/ports.conf:224]:
```suggestion
sysutils/puppet7					arm,arm64
```

File: config/21.1/plugins.conf [hunk 2]
```diff
@@ -69,6 +69,7 @@ sysutils/lcdproc-sdeclcd			arm,arm64
 sysutils/munin-node				arm,arm64
 sysutils/node_exporter				arm,arm64
 sysutils/nut					arm,arm64
+sysutils/puppet-agent
```
Comment (fichtner) [inline config/21.1/plugins.conf:72]:
```suggestion
sysutils/puppet-agent				arm,arm64
```

**Comment by fichtner** [comment]
I'm getting a problematic error message here for the build during final puppet7 build:

    Could not load facter; cannot install

Have you guys ever seen this?

**Comment by fichtner** [comment]
I think I found out why, geez 🤦

**Comment by fichtner** [comment]
Adds about 10 MB to the packages and builds in 1-2 minutes so this is ok, thanks!

**Comment by fichtner** [comment]
BTW, I'm bumping Ruby to 2.7 as per current FreeBSD ports default.

**Pull request merged**